ckanext/honduras/dataset.json

```diff
@@ -0,0 +1,118 @@
+{
+  "scheming_version": 1,
+  "dataset_type": "dataset",
+  "about": "Honduras CKAN dataset schema",
+  "about_url": "http://github.com/datopian/ckanext-honduras",
+  "dataset_fields": [
+    {
+      "field_name": "title",
+      "label": "Nombre del Dataset",
+      "required": true,
+      "preset": "title"
+    },
+    {
+      "field_name": "description",
+      "label": "Descripción del dataset",
+      "required": true
+    },
+    {
+      "field_name": "identifier",
+      "label": "Identificador del Dataset",
+      "required": true
+    },
+    {
+      "field_name": "theme",
+      "label": "Temática del dataset",
+      "required": true
+    },
+    {
+      "field_name": "keyword",
+      "label": "Palabras clave",
+      "required": true
+    },
+    {
+      "field_name": "issued",
+      "label": "Fecha de Creación del Dataset"
+    },
+    {
+      "field_name": "modified",
+      "label": "Fecha de última actualización del Dataset",
+      "required": true
+    },
+    {
+      "field_name": "accrualPeriodicity",
```

Comment:
this should be a [`select` preset](https://github.com/ckan/ckanext-scheming/#choices). The `choices` are in the "VARIABLES" sheet of the spreadsheet (Column "Frecuencia de actualización de la información"). Use the values on the spreadsheet as labels, and for values the lower-case, hyphenated form of them (also replacing accents like í -> i)